crates/bevy_input/src/gamepad.rs

```diff
@@ -276,6 +276,9 @@ pub enum GamepadButtonType {
     DPadLeft,
     /// The right button of the D-Pad.
     DPadRight,
+
+    /// Miscellaneous buttons, considered non-standard (i.e. Extra buttons on a flight stick that do not have a gamepad equivalent).
+    Button(u8),
```

Comment:
Why not name that one `Other` like the axis one? It's a little confusing, as the other values in the enum are also buttons.

Reply:
if that makes more sense then that's totally fine, will switch it over.